vuu-ui/packages/vuu-layout/src/layout-action.ts

```diff
@@ -1,21 +1,21 @@
-export const Action = {
-  ADD: 'add',
-  BLUR: 'blur',
-  BLUR_SPLITTER: 'blur-splitter',
-  DRAG_START: 'drag-start',
-  DRAG_STARTED: 'drag-started',
-  DRAG_DROP: 'drag-drop',
-  FOCUS: 'focus',
-  FOCUS_SPLITTER: 'focus-splitter',
-  INITIALIZE: 'initialize',
-  MAXIMIZE: 'maximize',
-  MINIMIZE: 'minimize',
-  REMOVE: 'remove',
-  REPLACE: 'replace',
-  RESTORE: 'restore',
-  SAVE: 'save',
-  SET_TITLE: 'set-title',
-  SPLITTER_RESIZE: 'splitter-resize',
-  SWITCH_TAB: 'switch-tab',
-  TEAR_OUT: 'tear-out'
-};
+export enum Action {
```

Comment:
I would probably have left this as it was, my understanding is that typescript enums should be avoided if you can.

Reply:
will look into this further, i've not heard of any issues and quite like them but if consensus is against using then i'll drop it

Reply:
Yeah I can't get behind the "transpiled enums are ugly" argument but the typesafety stuff makes sense to me, I'll try implementing this as a union type

Reply:
Done